cypress/e2e/rooms/changeRoomsPermission/changeRoomPermissionViewerToAdmin.feature

```diff
@@ -0,0 +1,99 @@
+        When I enter '<participant_name>' in name dropdown
+        When I select the first name from the dropdown
+        When I click on the button to add the participant
+        Then I see '<participant_name>' in the room participants list
+
+        # first teacher changes role for other user
+        When I click on button Three Dot Menu to add participant '<participant_name>'
+        Then I see button Change Role Permission is visible
+        When I click on button 'Change-Permission' in the sub-menu
+        Then I see dialog box Change Role Permission is visible
+        When I change second user role to 'admin'
+        Then I see Role changed to 'admin' for second user
+        Then I click on button 'Confirm' in the action menu
+
+        # second user logs in, accesses the room, verifies that restricted functionalities are not visible to them, and then leaves the room
+        Given I am logged in as a '<teacher_2>' at '<namespace>'
+        When I go to room overview
+        Then I see '<room_name>' on room overview page
+        When I go to room '<room_name>'
+        Then I see the detail page of room '<room_name>'
+        Then I see button Fab Create Room Board
+        When I click on three dot menu in room page
+        Then I don't see 'delete' options in the menu
+        When I select the three dot menu action 'edit'
+        When I enter the room name '<room_name_edited>'
+        When I click on the button to save the room
+        Then I see the detail page of room '<room_name_edited>'
+        When I click on three dot menu in room page
+        When I select the three dot menu action 'room-members'
+        Then I see the edit participants page of room '<room_name_edited>'
+        Then I see button Fab Add Member
+        Then I see info text
+        Then I see first checkbox column in the table
+        Then I see last actions column in the table
+        When I go to room overview
+        Then I see '<room_name_edited>' on room overview page
+        When I go to room '<room_name_edited>'
+        When I click on three dot menu in room page
+        When I select the three dot menu action 'leave-room'
+        Then I see dialog box to leave the room
+        Then I click on button 'Confirm' to leave the room
+
+        # first teacher logged in and assert second user is not in the table
+        Given I am logged in as a '<teacher_1>' at '<namespace>'
+        When I go to room overview
+        Then I see '<room_name_edited>' on room overview page
+        When I go to room '<room_name_edited>'
+        Then I see the detail page of room '<room_name_edited>'
+        When I click on three dot menu in room page
+        When I select the three dot menu action 'room-members'
+        Then I see the edit participants page of room '<room_name_edited>'
+        Then I see teacher '<participant_name>' not visible in the table
+
+        # post-condition: first teacher deletes the room
```

Comment:
also here you can use a postcondition method from PrePostConditionSteps

Reply:
done